checkout-ui-custom/src/controller/DeliverController.js

```diff
@@ -169,6 +169,16 @@ const DeliverController = (() => {
     }
   });
 
+  // address type - change building/complex label to either business
+  $(document).on('change', $('input').attr('addressType'), () => {
```

Comment:
Hey @SegopotsoMakhutja I think you mean this?
```
  $(document).on('change', 'input[name="addressType"]', function() {
```
Also, in case you need to know for future, with `function()` we have access to `this`, so you could do things like `$(this).is(":checked")`  Not necessary in this case though - the logic you have below should work fine.

Reply:
Hey @simmbiote thanks, i've updated.